Dockerfile

```diff
@@ -6,7 +6,10 @@ ARG DEBIAN_FRONTEND=noninteractive
 
 # Set the user ID to the default value of 1001 since different ubuntu
 # versions yield to different user IDs with `useradd`
-ENV USER_ID=1001
+ENV USER_ID=1000
+# Delete the default ubuntu user and group in Ubuntu 23.04+
+# that conflicts with linuxbrew i.e UID=1000 GID=1000
+RUN { touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu; } || true
```

Comment:
```suggestion
RUN touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu || true
```

would this work?

Reply:
That was my initial version. Failed https://github.com/Homebrew/brew/actions/runs/9854447471/job/27207197218

Reply:
```suggestion
# shellcheck disable=SC2015
RUN touch /var/mail/ubuntu && chown ubuntu /var/mail/ubuntu && userdel -r ubuntu || true
```